.github/workflows/ci.yml

```diff
@@ -1,7 +1,6 @@
 name: CI
 on:
   push:
-    branches: ['**']
```

Comment:
is it safe to remove it?

Reply:
It's duplicating the job executions after we added the `pull_request`.
My understanding is that we should choose which one we want to keep, for every branch (and it includes pull requests) or only when a pull request is created.